examples/SPDXYAMLExample-2.2.spdx.yaml

```diff
@@ -1,392 +1,390 @@
-      \ OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN\
-      \ NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,\
-      \ SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,\
-      \ PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;\
-      \ OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER\
-      \ IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)\
-      \ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE\
-      \ POSSIBILITY OF SUCH DAMAGE."
-    licenseId: "LicenseRef-2"
-  - extractedText: "/*\n * (c) Copyright 2000, 2001, 2002, 2003, 2004, 2005, 2006,\
-      \ 2007, 2008, 2009 Hewlett-Packard Development Company, LP\n * All rights reserved.\n\
-      \ *\n * Redistribution and use in source and binary forms, with or without\n\
-      \ * modification, are permitted provided that the following conditions\n * are\
-      \ met:\n * 1. Redistributions of source code must retain the above copyright\n\
-      \ *    notice, this list of conditions and the following disclaimer.\n * 2.\
-      \ Redistributions in binary form must reproduce the above copyright\n *    notice,\
-      \ this list of conditions and the following disclaimer in the\n *    documentation\
-      \ and/or other materials provided with the distribution.\n * 3. The name of\
-      \ the author may not be used to endorse or promote products\n *    derived from\
-      \ this software without specific prior written permission.\n *\n * THIS SOFTWARE\
-      \ IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR\n * IMPLIED WARRANTIES,\
-      \ INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES\n * OF MERCHANTABILITY\
-      \ AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.\n * IN NO EVENT SHALL\
-      \ THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT,\n * INCIDENTAL, SPECIAL, EXEMPLARY,\
-      \ OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT\n * NOT LIMITED TO, PROCUREMENT OF\
-      \ SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,\n * DATA, OR PROFITS; OR BUSINESS\
-      \ INTERRUPTION) HOWEVER CAUSED AND ON ANY\n * THEORY OF LIABILITY, WHETHER IN\
-      \ CONTRACT, STRICT LIABILITY, OR TORT\n * (INCLUDING NEGLIGENCE OR OTHERWISE)\
-      \ ARISING IN ANY WAY OUT OF THE USE OF\n * THIS SOFTWARE, EVEN IF ADVISED OF\
-      \ THE POSSIBILITY OF SUCH DAMAGE.\n*/"
-    licenseId: "LicenseRef-1"
+SPDXID: "SPDXRef-DOCUMENT"
+spdxVersion: "SPDX-2.2"
+creationInfo:
+  comment: "This package has been shipped in source and binary form.\nThe binaries\
+    \ were created with gcc 4.5.1 and expect to link to\ncompatible system run time\
+    \ libraries."
+  created: "2010-01-29T18:30:22Z"
+  creators:
+  - "Tool: LicenseFind-1.0"
+  - "Organization: ExampleCodeInspect ()"
+  - "Person: Jane Doe ()"
+  licenseListVersion: "3.8"
```

Comment:
Minor issue but the license list is now at 3.9 should we update `licenseListVersion` in the examples?

Reply:
Manually updated the license list to 3.9 (much faster than regenerating all the files).